### PR TITLE
[I18N] Update RU strings.xml

### PR DESCRIPTION
**Pull request by Pitstopper**
Some changes and additions to the translation.

### CODE REVIEW

File: app/src/main/res/values-ru-rRU/strings.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <string name="msg_delete_account">Удалить аккаунт</string>
     <string name="msg_change_status">Изменить статус</string>
 
-    <string-array name="languages"> <!-- TODO Add translations -->
+    <string-array name="языки">
```
Comment (Adizbek) [inline app/src/main/res/values-ru-rRU/strings.xml]: You shouldn't change name of array.
Write name of languages in Russian which listed below.

Comment (philipbrito) [inline app/src/main/res/values-ru-rRU/strings.xml]: @Pitstopper This is making the app not building. You need to translate the strings bellow (English, Arabic, German, etc) not the name of the `string-array`.

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Review by Pitstopper — COMMENTED** [review]
Fixes

**Review by Adizbek — APPROVED** [review]

**Comment by Pitstopper** [comment]
Sorry, my bad

File: app/src/main/res/values-ru-rRU/strings.xml (hunk 1 above)
Comment (Pitstopper) [reply app/src/main/res/values-ru-rRU/strings.xml]: Fixes made, checks passed

Comment (Pitstopper) [reply app/src/main/res/values-ru-rRU/strings.xml]: Fixes made

**Comment by philipbrito** [comment]
Don't worry @Pitstopper.  It happens ....
Thank you very much for always sending the translations for Russian language ❤️

**Review by philipbrito — APPROVED** [review]

**Pull request merged**